### DISCUSSION TITLE
Unable to detect roms on the SD card (multiple attempts)

### DISCUSSION THREAD

**Discussion by RedJoker7** [OnionUI/Onion#1293] [Support] [answered]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Docs](https://onionui.github.io/docs) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.2

### Miyoo Firmware Version

354FW

### Provide a clear and concise description of the issue

Hello, I formatted my SD card with GUI and successfully installed Onion OS 4.2 on my SD card. I also installed Roms and BIOS into the respective directories. However, when I hit on select and refresh roms. I seem to sometimes be able to load just one set of roms and the others are empty. Sometimes, it directly shows empty without detecting roms. Kindly help me out! please note that all Roms are on the correct format. 

### Steps to Reproduce

Steps

1. Load Roms and BIOS into the respective folders on SD card

2. Boot Miyoo Mini Plus -------> Highlight games tab ------> Select -------> Refresh Roms

3. Result - May detect on one console but not others, sometimes not detect and just shows as loading even after waiting over 10 mins.

### Provide any additional context or information that may help us investigate your issue

Onion OS was successfully installed, roms have been loaded with the correct format, all folders have been named correctly.

Example 

1. Roms is Roms not ROMS

2. All game folder are upper cased. Example : SFC is SFC

Also note that is a completely news SD card formatted to FAT32 with GUI and installed Onion OS 4,2 after.

Thank you for your time and patience! I hope my query can be solved!

**Comment by Phanturian** [comment]
Did you install the game systems using Package Manager before adding the roms? You mention "all folders have been named correctly" and "All game folders are upper cased" but when you install the systems via Package Manager, it generates the folders for you. No need to rename or mess with folder names. Then you add your roms to those folders and you should be good to go.

**Comment by RedJoker7** [comment, marked as answer]
Update : Got a new SD card. Did a fresh install of Onion OS. ROMs are now detectable and playable. Thank you guys so much for your assistance! let me know if I could support you guys in any way! I'd be glad to help! 😄 Cheers!! I love it!

**Reply by Levinbb** [reply]
Hi i have the same problem as you about formatting the sd card to onion os and now its empty. Im still using the stock sd card. Hope you can help thanks so much
